packages/console/package.json

```diff
@@ -100,5 +101,8 @@
       ]
     }
   },
-  "prettier": "@silverhand/eslint-config/.prettierrc"
+  "prettier": "@silverhand/eslint-config/.prettierrc",
+  "dependencies": {
+    "color": "^4.2.3"
+  }
```

Comment:
show be in dev dependencies

Reply:
They are used in https://github.com/logto-io/logto/blob/ac118ad50c1eb4d3529c65686523c8bd05e070ee/packages/console/src/pages/SignInExperience/components/ColorForm.tsx#L29

Although we can refactor `absoluteLighten` in `shared` to receive `string`, but this function is used elsewhere, and that won't reduce the bundle size.

Reply:
ah the `devDeps` is not for reduced bundle size. it's just for dependency management. we don't need these deps in prod since they'll be bundled into `/dist`

Reply:
Got it!

Reply:
PTAL